exporter.go

```diff
@@ -222,7 +223,8 @@ func performDataExportToS3(configuration *ConfigStruct,
 	// log into terminal
 	printTables(tableNames)
 
-	bucket := GetS3Configuration(configuration).Bucket
+	bucket := GetS3Configuration(configuration).Prefix + "/" + GetS3Configuration(configuration).Bucket
```

Comment:
WDYT about ability to have empty prefix, i.e. bucket path w/o `/` at the beginning?

Reply:
I thought we wouldn't want that, but since you're asking... that's a quick to make change =)

Reply:
done